### PR TITLE
Effectively expose RuleContributions.md for developers

### PR DESCRIPTION
**Pull request by shaopeng-gh**
RuleContributions.md is important to read, 
most people will not open the doc folder and open each one of the file to see which one meets the need, 
so it is important to link to in the home page of our repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +21,9 @@ This repository contains the source code for BinSkim, a Portable Executable (PE)
 3. Find out more about the Static Analysis Results Interchange Format **([SARIF](https://github.com/sarif-standard/sarif-spec/))** used to output Binskim results
 
 ### How to extract the exe file from the nuget package
```
Comment (shaopeng-gh) [inline README.md:23]: there two empty lines are done by the md file standard format, not manual change. I verified it looks the same from UI.

File: README.md [hunk 2]
```diff
@@ -5,8 +5,9 @@ This repository contains the source code for BinSkim, a Portable Executable (PE)
 ## For Developers
 
 1. Fork the repository -- **[Need Help?](https://help.github.com/articles/fork-a-repo/)**
```
Comment (michaelcfanning) [inline README.md:7]: >Fork the repository [](http://example.com/codeflow?start=3&length=19)

One thing, when we encourage users to fork the repository, we lead them to submit PRs that can't be validated!

Are we set up to allow people without any explicit permissions in the repo to create a branch off and submit a PR?

I can ask someone on the team to run a test.

Comment (michaelcfanning) [reply README.md:7]: Wait! Shaopeng, don't you have a different non-MS GitHub account? If so, can you try to enlist, then create a branch, then push that branch, then touch a file, and submit a PR?

The question is whether this works for you.

Comment (shaopeng-gh) [reply README.md:7]: behold :)
this is from a new account I just created and a touch of the file that is safe to merge in,
please click approve the workflow run and we can continue the flow to merge it in to verify this procedure works:
https://github.com/microsoft/binskim/pull/867

Comment (shaopeng-gh) [reply README.md:7]: oh I can approve, I have clicked that.

Comment (shaopeng-gh) [reply README.md:7]: I have left my comments in that PR. https://github.com/microsoft/binskim/pull/867

**Review by suvamM — APPROVED** [review]

**Comment by shaopeng-gh** [comment]
thanks @suvamM, I think Michael comments are about the overall process, the specific change in this PR should be safe to merge in.

**Pull request merged**